src/services/api.js

```diff
@@ -26,7 +26,7 @@ const getSplitValue = (key, raw, delim1, delim2) => {
 }
 
 export const deleteCookie = name => {
-  const domain = process.env.COOKIE_DOMAIN || window.location.hostname
+  const domain = window.location.hostname
```

Comment:
Can you walk me through the usage of this? This doesn't seem like it would work since the cookie would be scoped to backend; e.g., `api.eapp.example.com` and this will be executing in the frontend context?

Reply:
This code is removed in the FE branch, sorry for the confusion.

Reply:
No worries – under this revised paradigm, under what circumstances does the cookie get deleted?